### ISSUE TITLE
The appearance style of the Admin Plugin is incomplete after the upgrade

### ISSUE THREAD

**Issue by petira** [getgrav/grav-plugin-admin#2035]
The appearance style of the Admin Plugin is incomplete after the upgrade (bookmarks, buttons and other form elements are not displayed correctly).

So far, I have upgraded 2 sites, one went smoothly (from Grav v1.6.28), but the other (Grav v1.6.31) does not display the Admin Plugin correctly.

Grav v1.7.3 - Admin v1.10.2

**Comment by rhukster** [comment]
Could you please provide a screenshot ?

**Comment by petira** [comment]
Of course:
![example](https://user-images.githubusercontent.com/38700805/105511823-a0766a00-5cd0-11eb-957f-a1092f722066.png)

**Comment by rhukster** [comment]
Does this install happen to have any other admin ui type plugins? 

Also what happens if yo got plugins and admin and hit recompile ?

**Comment by petira** [comment]
**Installed plugins:**
1.  Admin Panel  v1.10.2
2.  Admin Power Tools v0.1.17 (Disabled)
3.  Archives  v2.0.2
4.  Breadcrumbs  v1.6.1
5.  Core Service Manager v1.1.4 (Disabled)
6.  Editor v0.1.12 (Disabled)
7.  Email  v3.1.0
8.  Error  v1.7.1
9.  Flex Objects  v1.0.1
10.  Form  v4.3.0
11.  Imagecreate v0.2.0
12.  Login  v3.3.8
13.  Markdown Notices  v1.1.0
14.  Pagination  v1.4.2
15.  Problems  v2.0.3
16.  Random  v1.5.0
17.  Related Pages  v1.2.2
18.  Shortcode Core  v5.0.3
19.  Sitemap  v2.0.2
20.  Stamp v1.0.1
21.  Taxonomy List  v1.3.5

**Recompile:**
- No changes after Recompile.

**Comment by rhukster** [comment]
Can you try disabling the non-core plugins (eg imagecreate and stamp). I don't know what they do.  Also have you tried another browser in case it's browser cache thing?

**Comment by petira** [comment]
I disabled plugins (Imagecreate and Stamp), but to no avail. I tried different browsers, but also to no avail. Of course, I also tried to delete the browser cache.

**Comment by petira** [comment]
It is also interesting that if it loads after the login as the first Dashboard, it will be displayed normally. But once I switch to another section, everything falls apart.

**Comment by petira** [comment]
@rhukster I checked the source code of the same page when it is displayed correctly and when it is displayed incorrectly, and they are completely identical. The only difference is in the `'?id'` parts, for example:

```
...
<link href="/user/plugins/admin/themes/grav/css/iconpicker.css?XXX" type="text/css" rel="stylesheet">
...
<script src="/system/assets/jquery/jquery-2.x.min.js?XXX"></script>
...
<input type="hidden" name="admin-nonce" value="YYY" />
...
```
I also noticed that when loading pages, there is a kind of loop that prevents the page from applying styles, at least I think so.

**Comment by petira** [comment]
@rhukster Following the previous.

The problem is really related to style `/assets/admin-preset.css`, because it is deleted uncontrollably every time you switch to another section.

Re-creation occurs when you log out of the system (after the subsequent login, the first page is displayed correctly) or `Recompile` button is pressed (style is only created but does not affect the display) or `Preview` button is pressed (style is created and the current page `Admin Panel` is displayed correctly).

The question remains why this happens on this site, when on another site that has the same settings, everything is fine?

**Comment by rhukster** [comment]
Are you sure the file is being deleted in those instances? When it's not displaying, can you check the filesystem and see if that file exists.  More likely it's not being included for whatever reason.  I've not seen this before and it's definitely not widespread so not sure how to recreate it.  Perhaps you can zip up your site and make it available for me to download?

**Comment by petira** [comment]
Yes, I checked it via a URL first, and then verified the above behavior on FTP.

The permissions on both sites are the same. On the first site is the original css file after installation, on the second the file disappears and reappears with a new date.

If I subsequently sent you a website, do you want it complete, or just a specific part?

**Comment by rhukster** [comment]
The entire user/ folder should be fine.

**Comment by petira** [comment]
@rhukster, @iusvar I found out the cause of the problem. This is caused by the **Imagecreate** plugin. Disable is not enough, but it must be completely removed.

**Comment by rhukster** [comment]
Added to the docs.  Thanks https://learn.getgrav.org/17/advanced/grav-development/grav-17-upgrade-guide#missing-css-styling-in-admin

**Comment by petira** [comment]
@rhukster As I wrote, disable the plugin is not enough, but the plugin must be completely removed.

@iusvar Can you update the Imagecreate plugin to be compatible with Admin plugin 1.10, please? The frontend is fully compatible with Grav 1.7 now.

**Comment by rhukster** [comment]
I will update my learn entry to reflect that @petira .

**Comment by rhukster** [comment]
For now i'm going to have to recommend completely removing the plugin.  When the plugin is updated to support Admin 1.10 I'll adjust the docs to just recommend updating.

**Comment by AwebsomeFr** [comment]
Hi,

I have the same issue with grav 1.7 and grav-admin v1.10.3, but I don't have any installed plugin like imagecreate. 

I updated plugins one by one. Everything was OK until I updated the plugin admin (in last).

I tested the interface on Firefox and Chrome (with refreshing the cache). With no result. Some styles are missing, I guess. And it seems that the viewport is fixed to 100vh, so everything is beyond is hidden (no scroll).

If it can help, here the full list on my plugins :
- Admin Panel  - v1.10.3  
- DevTools  - v1.4.2  
- Email - v3.1.1  
- Error - v1.7.1
- Flex Objects v1.0.2   
- Form v4.3.1
- Login  v3.4.0  
- Problems v2.0.3

Any idea ?
Thanks !
